cdist/conf/type/__consul_template/files/consul-template.sysv

```diff
@@ -10,21 +10,23 @@
 # pidfile: /var/run/consul-template/pidfile
 
 # Source function library.
+# shellcheck disable=SC1091
 . /etc/init.d/functions
 NAME=consul-template
 CONSUL_TEMPLATE=/usr/local/bin/consul-template
 CONFIG=/etc/$NAME/conf.d
 PID_FILE=/var/run/$NAME/pidfile
 LOG_FILE=/var/log/$NAME
 
+# shellcheck disable=SC1090
 [ -e /etc/sysconfig/$NAME ] && . /etc/sysconfig/$NAME
 export CONSUL_TEMPLATE_LOG=${CONSUL_TEMPLATE_LOG:-info}
 export GOMAXPROCS=${GOMAXPROCS:-2}
 
 mkdir -p /var/run/$NAME
 
 start() {
-   echo -n "Starting $NAME: "
+   printf "Starting %s" "$NAME: "
```

Comment:
nitpick: the trailing space would be better in the formatting part instead of the argument.